docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml

```diff
@@ -2281,23 +2281,32 @@ HPCCPrivateKeyFile=/keyfilepath/keyfile
         number of cores divided by two is the maximum number of Thor clusters
         to use.</para>
 
-        <sect3>
-          <title>Multiple Nodes</title>
-
-          <para>Try to keep resources running on their own nodes, if possible
-          for either one or multiple Thor clusters. If running some kind of
-          active/passive high availability, don't keep your active and passive
-          master on the same node. Try to keep Dali and ESP on separate nodes.
-          Even if you don't have the luxury of very many nodes, you want the
-          Thor master and the Dali (at minimum) to be on separate nodes. The
-          best practice is to keep as many components as possible on their own
-          nodes.</para>
+        <sect3 id="SepClusterNodes">
+          <title>Separate Target Cluster Nodes</title>
+
+          <para>Try as much as possible to keep system target cluster
+          resources running on their own nodes. The idea being to avoid
+          putting essential system components on the same partitions as
+          highly-variable cluster data. Another condition could be, If you are
+          running some kind of active/passive high availability, don't keep
+          your active and passive master on the same node. Try to keep Dali
+          and ESP on separate nodes. Even if you don't have the luxury of very
+          many nodes, you would want the Thor master and the Dali (at a
+          minimum) to be on separate nodes. Thor can have varying and
+          expansive data sizes and is not uncommon to have that data take up
+          most of the available capacity on that partition, or even all of it.
```

Comment:
>you would want the Thor master and the Dali (at a
          minimum) to be on separate nodes. Thor can have varying and
          expansive data sizes and is not uncommon to have that data take up
          most of the available capacity on that partition, or even all of it.


Thor master is the one part of Thor that does not take up disk space...
The Thor slaves create physical files/consume disk space, so this section should be reworded.


Perhaps a separate issue, but this doc refers to Master and Slave's - have other docs. moved to Manager and Workers?

Reply:
we have replaced many of them - Jim also commented on the same above.